app/controllers/ldp/ReadWriteWebControllerGeneric.scala

```diff
@@ -79,7 +79,7 @@ trait ReadWriteWebControllerGeneric extends ReadWriteWebControllerTrait {
   private def allowHeaders(authResult: AuthResult[NamedResource[Rdf]]): List[(String, String)] = {
     val modesAllowed = authResult.authInfo.modesAllowed
     val isLDPC =  authResult.result.isInstanceOf[LocalLDPC[_]]
-    val allow = "Allow" -> {
+    val allow = "Access-Control-Allow-Methods" -> {
```

Comment:
The LDP spec requires the Allow. See [¶ 4.2.8.2 of the LDP spec](http://www.w3.org/TR/ldp/#ldpr-resource). So it can't be replaced. 
The [Access-Control-Allow-Methods is part of the CORS standard](http://www.w3.org/TR/cors/#access-control-allow-methods-response-header) and has a different meaning. The reason for its existence has to be looked at in a lot greater detail to understand what the dangers associated with it are.